### ISSUE TITLE
Using custom matchers

### ISSUE THREAD

**Issue by Ginja** [burtlo/ark#105]
This is probably an issue of me not understanding how to use custom matchers, but if I were to use and depend on this cookbook in my wrapper cookbook, should I not be able to use the custom matchers you've defined in `ark/libraries/matchers.rb`? I found I needed to use something like this:

https://github.com/lynx44/chefspec_extensions

**Comment by burtlo** [comment]
I would think that you should be able to use it. 

Maybe its because I haven't released a version of the cookbook with the chefspec helpers defined within it?

**Comment by burtlo** [comment]
Yeah I found it works when I update the Berksfile to use ark from github.

```
source "https://supermarket.chef.io"

metadata

cookbook "ark", git: "git@github.com:burtlo/ark.git"
```

**Comment by dhoer** [comment]
Matchers are usually included in with the cookbook. @burtlo, can the next release include the matchers?

**Comment by dwmarshall** [comment]
This is fixed with ark v1.0.0
